app.js

```diff
@@ -0,0 +1,36 @@
+var express = require('express');
+var path = require('path');
+var bodyParser = require('body-parser');
+var md5 = require('md5');
+var multer = require('multer');
+var app = express();
+var port = process.env.port || 3003; 
+
+var storage = multer.diskStorage({
+	destination:function(req,file,callback) {
+		callback(null,'./uploads/user');
+
+	},
+	filename:function(req,file,callback) {
+		var fileUniqueName = md5(Date.now());
+		 callback(null,  fileUniqueName + path.extname(file.originalname));
+	}
+});
+var upload = multer({storage:storage});
+
+app.use(bodyParser.urlencoded({
+	extended: true
+}));
+app.use(bodyParser.json());
+
+var user_panel = require('./routes/user');
+
+app.use(express.static(path.join(__dirname, 'views')));
+
+app.post('/user/login', user_panel.login);
+app.post('/user/signup', user_panel.signup);
+app.post('/user/update',upload.any(), user_panel.update);
```

Comment:
if we update the profile image, then we don't require upload.any()

Reply:
we are uploading profile image and cover image of a user at a time so we use upload.any().

Reply:
for multiple image uploading we use upload.any()